package.json

```diff
@@ -15,6 +17,9 @@
     "type": "git",
     "url": "git+https://github.com/fieldguide/pipeline.git"
   },
+  "files": [
+    "build/"
```

Comment:
Minor, but this contains the compiled tests at the moment. I'd recommend the following:

1. Create a separate TSConfig for pre-publishing, i.e. `tsconfig.pack.json`:

    ```json
    {
      "extends": "./tsconfig.json",
      "exclude": ["**/__mocks__", "**/__tests__"]
    }
    ```

1. Update the build script to target that configuration:

    ```bash
    tsc --project tsconfig.pack.json
    ```

Reply:
This is great, and included in 1b12284. I used `tsconfig.build.json` to match the main command name. To me, `pack` feels like a very npm-kind-of-thing. `build` feels more generalized. So the mental model I took is that the npm `prepack` stage should run the `build`.

Super small, and at the end of the day it has little consequence. I don't know why I have opinions about these kinds of things, but I do. 😆

Reply:
Yeah, "build" is a better word 😅